_specifications/lsp/3.18/specification.md

```diff
@@ -61,7 +61,9 @@ Content-Length: ...\r\n
 ```
 ### Base Protocol JSON structures
 
-The following TypeScript definitions describe the base [JSON-RPC protocol](http://www.jsonrpc.org/specification):
```

Comment:
Could we change this to include `currently`. There are feature requests to support batching.